packages/flame/lib/src/geometry/shape.dart

```diff
@@ -10,7 +10,7 @@ import 'shape_intersections.dart' as intersection_system;
 /// A point can be determined to be within of outside of a shape.
 abstract class Shape {
   /// The position of your shape, it is up to you how you treat this
-  Vector2 position = Vector2.zero();
+  late Vector2 position;
```

Comment:
if it's set in the constructor initializer it doesn't even need to be late right? can just be final

Reply:
or just nothing (doesn't need to be final)

Reply:
Hmm true, but I doubt it can be final as I suspect it can be set again

Reply:
```suggestion
 Vector2 position;
```